optimum/utils/testing_utils.py

```diff
@@ -4,6 +4,32 @@
 import sys
 import unittest
 
+from packaging import version
+
+from optimum.utils import is_accelerate_available
+
+
+def require_accelerate(test_case):
+    """
+    Decorator marking a test that requires accelerate. These tests are skipped when accelerate isn't installed.
+    """
+    return unittest.skipUnless(is_accelerate_available(), "test requires accelerate")(test_case)
+
+
+def is_torch_greater_than_113():
```

Comment:
In `optimum/utils/version_utils.py`

Reply:
Same as https://github.com/huggingface/optimum/pull/423#discussion_r1018914347